account_fiscal_position_vies_warning/models/account_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Moduon Team S.L. <info@moduon.team>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def action_post(self):
+        if not self.env.company.vat_check_vies:
+            return super().action_post()
+        for record in self:
+            if (
+                record.fiscal_position_id
+                in self.env.company.check_vies_fiscal_position_ids
+                and not record.partner_id.vies_valid
+            ):
+                self.partner_id.raise_vies_warning(
+                    self.partner_id.name, record.fiscal_position_id.name
+                )
```

Comment:
Careful here, `self` is the recordset. You want to call the method over the current `record` in your loop
```suggestion
                record.partner_id.raise_vies_warning(record.fiscal_position_id)
```

Reply:
Thanks, that’s a great idea :+1:  I’ve updated this part of the code